eg/proximity.js

```diff
@@ -3,49 +3,14 @@ var five = require("../lib/johnny-five.js"),
 
 five.Board().on("ready", function() {
 
-  // Create a new IR Proximity Sensor hardware instance.
-  //
-  // five.IR.Proximity();
-  //
-  // (Alias of:
-  //   new five.IR({
-  //    device: "GP2Y0D805Z0F",
-  //    freq: 50
-  //   });
-  // )
-  //
-
-  prox = new five.IR.Proximity();
-  led = new five.Led(13);
-
-
-
-
-
-  // Properties
-
-  // prox.state
-  //
-
-  // Proximity Event API
-
-  // "motionstart"
-  //
-  // Fired motion is detected with 2"
-  //
-
-
-  // "motionend"
-  //
-  // Fired following a "motionstart" event
-  // when no movement has occurred in X ms
-  //
-  prox.on("motionstart", function(err, timestamp) {
-    led.on();
+  var proximity = new five.Proximity({
+    pin: "A0",
+    controller: "OA41SK",
+    freq: 250
   });
 
-  prox.on("motionend", function(err, timestamp) {
-    led.off();
+  proximity.on("data", function(err, data) {
```

Comment:
For events, we're trying to get rid of the `err, data` arguments. Update the class, tests, etc. to emit only a `data` object